pages/admin/invite/userAccountPage/[id].tsx

```diff
@@ -62,14 +315,14 @@ const UserAccountPage: React.FunctionComponent<UserRequest> = ({
         method: "PATCH",
         headers: { "Content-Type": "application/json" },
         credentials: "include",
-        body: JSON.stringify(({
+        body: JSON.stringify({
           status: UserStatus.Active,
```

Comment:
you have to add the added roles for selectedPlayers here so that you can update the user role with the new linked players